user/serializers.py

```diff
@@ -22,3 +23,42 @@ def update(self, instance, validated_data):
             user.save()
 
         return user
+
+
+class AuthTokenSerializer(serializers.Serializer):
+    class AuthTokenSerializer(serializers.Serializer):
```

Comment:
The `AuthTokenSerializer` class is defined twice. Remove the redundant definition to avoid confusion and potential errors.